db/migrate/20220105085730_migrate_customers_data.rb

```diff
@@ -11,14 +11,47 @@ def up
   end
 
   def migrate_customer_name_data!
-    Customer.where(first_name: "", last_name: "").where.not(name: [nil, ""]).find_each do |customer|
-      name_words = customer.name.split(' ')
-      next if name_words.empty?
-
-      customer.update(
-        first_name: name_words.first,
-        last_name: name_words[1..].join(' ')
-      )
+    customers_with_bill_addresses.find_each do |customer|
```

Comment:
Yes, that's what I thought we could do and then Lynne suggested that we don't need this. But okay, it's done now. Nice work.